src/librepgp/stream-parse.cpp

```diff
@@ -46,6 +48,7 @@
 #include "crypto/signatures.h"
 #include "fingerprint.h"
 #include "pgp-key.h"
+#include "ffi-priv-types.h"
```

Comment:
We should not access this and `<rnp/rnp.h>` headers here, probably it was forgotten to be removed.